manifest.json

```diff
@@ -6,7 +6,7 @@
         {
           "platforms": [
             {
-              "dockerfile": "src/alpine/3.6",
+              "dockerfile": "src/alpine/3.6/x64/base",
```

Comment:
amd64 is a better choice here as it aligns with what docker reports as the architecture.  It would make the convention consistent with the arm structure.